Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -388,14 +388,6 @@ static ASCIILiteral runningBoardDomainForAssertionType(ProcessAssertionType asse
     if (process.extensionProcess()) {
         ASCIILiteral runningBoardAssertionName = runningBoardNameForAssertionType(m_assertionType);
         ASCIILiteral runningBoardDomain = runningBoardDomainForAssertionType(m_assertionType);
-        auto didInvalidateBlock = [weakThis = ThreadSafeWeakPtr { *this }, runningBoardAssertionName] () {
```

Comment:
Why this change?

Reply:
Presumably so the code can be shared with the call on L477

Reply:
Yes, that is correct.

Thanks for reviewing, all!